cypress/cypress/integration/record.spec.ts

```diff
@@ -2078,6 +2080,96 @@ describe("Record", () => {
     cy.get("[data-cy=undo-question-btn]").should("be.disabled");
   });
 
+  it("Verify that transcript markdown loads as rich text: Q1", () => {
+    cyMockDefault(cy, {
+      mentor: clintMarkdown,
+      questions: chatQuestions,
+      gqlQueries: [
+        mockGQL("UploadTaskDelete", { me: { uploadTaskDelete: true } }),
+        mockGQL("UpdateAnswer", { me: { updateAnswer: true } }),
+        mockGQL("UpdateQuestion", { me: { updateQuestion: true } }),
+        mockGQL("ImportTask", { importTask: null }),
+        mockGQL("FetchUploadTasks", [{ me: { uploadTasks: [] } }]),
+      ],
+    });
+    cy.visit("/record?videoId=A1_1_1");
+    cy.get("[data-text]").should("be.visible");
```

Comment:
Does only checking if the text is visible really test if it's rich text? The test should probably check that the text isn't displaying any explicity markdown (asterisks, underscores, etc.)